errors/v2.yml

```diff
@@ -348,6 +348,11 @@
   http_code: 400
   message: "The service broker reported an error during provisioning: %s"
 
+60031:
```

Comment:
I think we've been trying to not add more errors in the v2.yml file? I think the proposed errors refactor are introducing some "generic" errors that can be reused. Maybe we can use the "UnableToPerform" error instead of this one? See https://github.com/cloudfoundry/cloud_controller_ng/blob/b9d9f1a8adb01a0165fc2279e3ff10c8af55b349/errors/v2.yml#L71